lib/app.jsx

```diff
@@ -489,7 +522,10 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 								selectedTag={state.tag}
 								tags={state.tags}
 								onOutsideClick={this.onToolbarOutsideClick} />
-							<div className="source-list theme-color-bg theme-color-fg">
+							<div className="source-list theme-color-bg theme-color-fg" ref={ ( ref ) => {
+								if ( !ref ) return;
+								ref.style.flexBasis = sourceList.width + 'px';
```

Comment:
Just realized you are already using this form. If you want to hold the reference, just save it on `this`